src/include/connection.h

```diff
@@ -347,8 +347,27 @@ struct __wt_connection_impl {
      * array when only a few threads are running.
      */
     WT_SESSION_IMPL *sessions; /* Session reference */
-    uint32_t session_size;     /* Session array size */
-    uint32_t session_cnt;      /* Session count */
+
+#define WT_CONNECTION_FOREACH_SESSION(session, conn)                                             \
```

Comment:
I would rename this to `WT_CONNECTION_SESSION_FOREACH`

Reply:
Renamed in [dc3766a](https://github.com/wiredtiger/wiredtiger/pull/9295/commits/dc3766ab2da4d497b526de5fbbb8063e091bcf3c)